WolvenKit/Views/Documents/RDTGraphView.xaml.cs

```diff
@@ -23,6 +24,28 @@ public partial class RDTGraphView : ReactiveUserControl<RDTGraphViewModel>
         public RDTGraphView()
         {
             InitializeComponent();
+
+            EventManager.RegisterClassHandler(typeof(BaseConnection), MouseLeftButtonDownEvent, new MouseButtonEventHandler(OnConnectionInteraction));
```

Comment:
There's a `SplitCommand` and `DisconnectCommand` on the connection that you can bind to in v2.0.0.